userspace/services.sh

```diff
@@ -60,4 +61,15 @@ systemctl mask systemd-backlight@.service
 systemctl disable rpcbind
 systemctl disable nfs-client.target
 systemctl disable remote-fs-pre.target
-systemctl disable run-rpc_pipefs.mount
+# Failed to disable unit, unit run-rpc_pipefs.mount does not exist.
+# systemctl disable run-rpc_pipefs.mount
+
+# Disable nvmf service since no NVMe-oF in the old kernel
+# this service fails in 24.04, while failing silently on 20.04
+# no influence on C3 NVMe nor nvme smart-log, which work fine
+systemctl disable nvmf-autoconnect.service
+
+# Service is from ifupdown but ifupdown is managed by NetworkManager
+# networking service fails with "ifup: failed to bring up lo"
+# no influence on any interface, all interfaces work fine
+systemctl disable networking.service
```

Comment:
```suggestion
systemctl disable networking.service

systemctl disable console-setup.service
systemctl disable sfsconfig.service
```

Reply:
- console-setup works fine for me in 24.04 - will recheck
- sfsconfig is something that seems to fail since forever - not related to 24.04

I don’t want to touch anything not related to 24.04 in this PR.

I suggest doing separate PRs.

Reply:
Console setup fails for me because it can't write to read-only /etc/console-setup/